teachopencadd/talktorials/001_query_chembl/talktorial.ipynb

```diff
@@ -1072,7 +909,7 @@
    "source": [
     "Note: If these numbers are the same, it means the latest ChEMBL release is still ChEMBL 27. \n",
     "\n",
-    "You can check out the current ChEMBL release [here](https://chembl.gitbook.io/chembl-interface-documentation/downloads)."
+    "You can check out the current [ChEMBL](https://chembl.gitbook.io/chembl-interface-documentation/downloads) release."
```

Comment:
URLs are linked with meaningful words, instead of pasting the URL directly or linking words like `here`